assets/quick-order-form.css

```diff
@@ -132,12 +150,130 @@
   align-items: center;
 }
 
-.quick-order-form .variant-item__totals .loading__spinner {
-  padding-top: 1.5rem;
-}
-
 .product-item__inner .small-hide {
   display: flex;
   flex-direction: column;
   align-self: center;
 }
+
+.quick-order-form tr.variant-item:not(.product).changed-row {
+  background-color: rgba(var(--color-foreground), 0.04);
```

Comment:
This is the right opacity for the selected state. Just a small adjustment would make it more efficient. ([Quick video](https://screenshot.click/27-19-esowl-q22dj.mp4))

Reply:
I created an [issue](https://github.com/Shopify/dawn/issues/3129) for these two things you mentioned.